### PR TITLE
Current version of the update.

### PR DESCRIPTION
**Pull request by microbit-carlos**
v2-beta branch includes a more generic way to create the drop down menus, so a lot of the old code for doing that has been removed to use the generic way.
Since the translations.js file then become 2 functions I've moved it inside python-main.js

### CODE REVIEW

File: editor.html [hunk 1]
```diff
@@ -348,28 +348,27 @@ <h2>Flashing micro:bit</h2>
         <script src="static/js/lzma-d.js"></script>
         <script src="static/js/base64.js"></script>
         <script src="static/js/microbit-fs.umd.js"></script>
+        <script src="static/js/urlparser.js"></script>
+        <script src="lang/uppercase.js" id="lang"></script>
+        <script src="translation.js"></script>
```
Comment (microbit-carlos) [inline editor.html]: Forgot to remove this one.

File: editor.html [hunk 2]
```diff
@@ -348,28 +348,27 @@ <h2>Flashing micro:bit</h2>
         <script src="static/js/lzma-d.js"></script>
         <script src="static/js/base64.js"></script>
         <script src="static/js/microbit-fs.umd.js"></script>
+        <script src="static/js/urlparser.js"></script>
+        <script src="lang/uppercase.js" id="lang"></script>
```
Comment (microbit-carlos) [inline editor.html]: Used this version instead of en for testing, should be reverted back when ready.

**Comment by microbit-carlos** [comment]
Okay, the issue has been fixed, have a look at the PR and let me know what you think.
If you are happy with it you can merge it and it will update the upstream PR automatically.

**Comment by JakubLenski** [comment]
LGTM, merging to translations

**Pull request merged**